src/components/dropdownSearch/DropdownSearchActions.tsx

```diff
@@ -67,17 +85,73 @@ export const addDropdownSearch = (id: string, optionsDropdown: IDropdownOption[]
   },
 });
 
+export const addMultiSelectDropdownSearch = (id: string, optionsDropdown: IDropdownOption[] = []): IReduxAction<IOptionsDropdownSearchPayload> => ({
+  type: DropdownSearchActions.addMultiSelect,
+  payload: {
+    id,
+    optionsDropdown,
+    filterText: '',
```

Comment:
If you did reuse another action, setting `filterText` to an empty string would be alright, but in the case of a brand new action, I would simply reset `filterText` in the reducer instead.